packages/plugin-react/src/index.ts

```diff
@@ -184,10 +200,9 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
 
       config.plugins.forEach(
         (plugin) =>
-          (plugin.name === 'react-refresh' ||
-            plugin.name === 'vite:react-jsx') &&
```

Comment:
Should keep this check. It provides a smoother upgrade path for users of [vite-react-jsx](https://www.npmjs.com/package/vite-react-jsx).

Reply:
I just realize 'vite:react-jsx' here does not refer to the plugin in the current plugin, but refers to the one in your `vite-react-jsx` plugin.
I have changed 'vite:react-jsx' to 'vite:react-jsx-runtime-automic' to make it more clearer